lib/ReplaceOpenCLBuiltinPass.cpp

```diff
@@ -3492,6 +3495,13 @@ bool ReplaceOpenCLBuiltinPass::replaceAtomics(Function &F,
   });
 }
 
+bool ReplaceOpenCLBuiltinPass::replaceAtomicInit(Function &F) {
+  return replaceCallsWithValue(F, [&](CallInst *CI) {
+    return clspv::InsertSPIRVOp(CI, spv::OpStore, {}, CI->getType(),
+                                {CI->getArgOperand(0), CI->getArgOperand(1)});
```

Comment:
It was done as OpStore while thinking that this could give a better performance and avoiding the overhead of OpAtomicStore, but not sure if that would give a better performance and I think scoped-modification-order for this operation might make some problems. So, I changed it to OpAtomicStore.